bitnami/dataplatform-bp2/values.yaml

```
@@ -1170,71 +1289,70 @@ wavefront:
       enableRuntimeConfigs: true
       ## Can be used to add additional discovery rules
       ##
-      config:
+      #config:
```

Could you add a space?
```suggestion
      # config:
```